packages/maskbook/src/plugins/UnlockProtocol/base.ts

```diff
@@ -0,0 +1,15 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { pluginDescription, pluginIcon, pluginName } from './constants'
+
+export const base: Plugin.Shared.Definition = {
+    ID: pluginName,
+    icon: pluginIcon,
+    name: { fallback: pluginName },
+    description: { fallback: pluginDescription },
+    publisher: { name: { fallback: 'Unlock Protocol' }, link: 'https://unlock-protocol.com/' },
```

Comment:
Are you the official member of the Unlock Protocol? I think you should put your name here.